usage.md

```diff
@@ -55,3 +74,27 @@ We also considered the following way to override the automatic detection, but cu
   <p>You can also manually disable paragraphs using the data-content-paragraphs="false" config.</p>
 </div>
 ```
+
+## renuo-cms with rails
+
+
+The gem [renuo-cms-rails](https://github.com/renuo/renuo-cms-rails) provides simpler usage of the renuo-cms if you use it with rails.
+
+## Installation
```

Comment:
do we want also to provide a kind of 'step-to-step' instruction, what is needed to get it running?